src/test/java/com/github/fishio/gui/TestAchievementScreen.java

```diff
@@ -0,0 +1,31 @@
+package com.github.fishio.gui;
```

Comment:
If somebody has any ideas to why this class does not recognize GuiTest when extending, be my guest and explain it to me.